indispensable/src/commonMain/kotlin/at/asitplus/signum/indispensable/asn1/Asn1Elements.kt

```diff
@@ -235,26 +255,25 @@ sealed class Asn1Element(
 
     @Serializable
     @ConsistentCopyVisibility
-    data class Tag private constructor(
-        val tagValue: ULong, val encodedTagLength: Int,
+    data class Tag internal constructor(
```

Comment:
why was the visibility changed?

Reply:
we now use this ctor directly in Asn1Decoding.kt